.github/workflows/cpp-packaging.yml

```diff
@@ -34,6 +34,11 @@ jobs:
     name: log-inputs
     runs-on: ubuntu-latest
     steps:
+      - name: log if skipping integration tests
+        if: |
+          github.event.inputs.skipIntegrationTests != 0 && github.event.inputs.skipIntegrationTests != ''
```

Comment:
Should this be || ?

Reply:
It was logically correct, but I made it the same as the check below (with a !) so it's clear that it's testing the same condition.

Reply:
Or rather, should it be "0 or empty" instead of "not 0 and not empty"?

Reply:
er, my mistake, I'm confusing myself

this is correct